client/components/themes-list/index.jsx

```diff
@@ -148,70 +159,165 @@ function ThemeBlock( props ) {
 	);
 }
 
-function Empty( props ) {
-	const { wpOrgThemes, emptyContent, searchTerm, upsellCardDisplayed, translate } = props;
+function Options( { isFSEActive, recordTracksEvent, searchTerm, translate, upsellCardDisplayed } ) {
+	const isLoggedInShowcase = useSelector( isUserLoggedIn );
 	const selectedSite = useSelector( getSelectedSite );
-	const shouldUpgradeToInstallThemes = useSelector(
-		( state ) => ! siteHasFeature( state, selectedSite?.ID, FEATURE_INSTALL_THEMES )
+	const canInstallTheme = useSelector( ( state ) =>
+		siteHasFeature( state, selectedSite?.ID, FEATURE_INSTALL_THEMES )
 	);
+	const isAtomic = useSelector( ( state ) => isAtomicSite( state, selectedSite?.ID ) );
+	const sitePlan = selectedSite?.plan?.product_slug;
 
-	const matchingThemes = useMemo(
-		() =>
-			wpOrgThemes?.filter(
-				( wpOrgTheme ) =>
-					wpOrgTheme?.name?.toLowerCase() === searchTerm.toLowerCase() ||
-					wpOrgTheme?.id?.toLowerCase() === searchTerm.toLowerCase()
-			) || [],
-		[ wpOrgThemes, searchTerm ]
-	);
+	const options = [];
 
 	useEffect( () => {
-		if ( shouldUpgradeToInstallThemes && ! emptyContent ) {
-			upsellCardDisplayed( true );
-		} else {
-			upsellCardDisplayed( false );
-		}
+		upsellCardDisplayed( true );
 		return () => {
 			upsellCardDisplayed( false );
 		};
-	}, [ emptyContent, shouldUpgradeToInstallThemes, upsellCardDisplayed ] );
+	}, [ upsellCardDisplayed ] );
+
+	// Design your own theme / homepage.
+	if ( isLoggedInShowcase ) {
+		// This should start the Pattern Assembler ideally, but it's not ready yet for the
+		// logged-in showcase, so we use the site editor as a fallback.
+		if ( isFSEActive ) {
+			options.push( {
+				title: translate( 'Design your own' ),
+				icon: addTemplate,
+				description: translate( 'Jump right into the editor to design your homepage.' ),
+				onClick: () =>
+					recordTracksEvent( 'calypso_themeshowcase_more_options_design_homepage_click', {
+						site_plan: sitePlan,
+						search_term: searchTerm,
+						destination: 'site-editor',
+					} ),
+				url: `/site-editor/${ selectedSite.slug }`,
+				buttonText: translate( 'Open the editor' ),
+			} );
+		}
+	} else {
+		// This should also start the Pattern Assembler, which is currently in development for
+		// the logged-out showcase. Since there isn't any proper fallback for the meantime, we
+		// just don't include this option.
+	}
+
+	// Do it for me.
+	options.push( {
+		title: translate( 'Hire our team of experts to design one for you', {
+			comment:
+				'"One" means a theme in this context (i.e. "Hire our team of experts to design a theme for you")',
```

Comment:
IIRC, we chose to keep "one" vague as it can mean both "theme" and "site".
Probably not a big deal for translators, but we might want to mention this in the comment.
E.g.
```suggestion
				'"One" could mean "theme" or "site" in this context (i.e. "Hire our team of experts to design a theme for you")',
```